README.md

```diff
@@ -29,20 +46,19 @@ fn main() {
 
     loop {
         match results.recv() {
-            Ok(result) => {
-                match result {
-                    Idle{addr} => {
-                        error!("Idle Address {}.", addr);
-                    },
-                    Receive{addr, rtt} => {
-                        info!("Receive from Address {} in {:?}.", addr, rtt);
-                    }
+            Ok(result) => match result {
+                Idle { addr } => {
+                    error!("Idle Address {}.", addr);
+                }
+                Receive { addr, rtt } => {
+                    info!("Receive from Address {} in {:?}.", addr, rtt);
                 }
             },
             Err(_) => panic!("Worker threads disconnected before the solution was found!"),
         }
     }
 }
+
 ```
 
 Note a Pinger is initialized with two arguments: the maximum round trip time before an address is considered "idle" (2 seconds by default) and the size of the ping data packet (16 bytes by default).  To explicitly set these values Pinger would be initialized like so: ```Pinger::new(Some(3000 as u64), Some(24 as i32))```
```

Comment:
Can you also update the README here? Previously the payload size was set this way but with your PR we don't need to specify the i32 type anymore

Reply:
I'm too careless. Thank you for pointing out that I have already made changes. Please review again.